source/utilities.tex

```diff
@@ -2920,6 +2920,18 @@
 
 \rSec3[optional.ctor]{Constructors}
 
+\pnum
+The exposition-only alias template \exposid{converts-from-any-cvref}
+is used by some constructors for \tcode{optional}.
+\begin{codeblock}
+template<class T, class W>
+using @\exposid{converts-from-any-cvref}@  // \expos
+  = disjunction<is_constructible<T, W&>, is_constructible<T, W>,
```

Comment:
This should be a variable template or concept instead of an alias template (1) because it's easier to use that way, and (2) because you're already using it as if it is.
```suggestion
inline constexpr bool @\exposid{converts-from-any-cvref}@  // \expos
  = disjunction_v<is_constructible<T, W&>, is_constructible<T, W>,
```

Reply:
Oops, yes, it needed `::value` on the uses, thanks for catching that.